nix/app.nix

```diff
@@ -24,18 +24,18 @@ let
       in
       lib.cleanSourceWith { src = craneLib.path ./..; filter = isSourceFile; };
 
+    strictDeps = true;
```

Comment:
what's this do?

Reply:
It tells it not to get confused between `nativeBuildInputs` (build-time) and `buildInputs` (runtime).